cmd/virtual-workspaces/command/cmd.go

```diff
@@ -55,8 +55,8 @@ func NewCommand(ctx context.Context, errout io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "workspaces",
-		Short: "Launch workspaces virtual workspace apiserver",
-		Long:  "Start a virtual workspace apiserver to managing personal, shared or organization workspaces",
+		Short: "Launch virtual workspace apiserver",
+		Long:  "Start a virtual workspace apiserver to manage personal, shared or organization workspaces",
```

Comment:
```suggestion
		Short: "Launch virtual workspace apiservers",
		Long:  "Start virtual workspace apiserver for different purposes.",
```

Reply:
That should future-proof it for a while :)

Reply:
not sure this is so much better. It runs these https://github.com/kcp-dev/kcp/tree/main/pkg/virtual: syncer, workspaces, apiexport, initializingworkspaces. More to come.

Reply:
@sttts what would you think about making the `Long` description say something about starting the root API server to be a bit more specific?

Reply:
Fine with me. Wondering whether we can get the list easily somehow.

Reply:
It's probably hidden deep in the constructor code.

Reply:
> Wondering whether we can get the list easily somehow.

Seems like a great thing to do in a codegen step 👍🏻 I'll update to just mention starting the root API server for now -- happy to open an issue tracking automating listing components in usage if that would be desirable

Reply:
:+1: to both